sql/synth_recipes.sql

```diff
@@ -954,26 +954,26 @@ INSERT INTO `synth_recipes` VALUES (13023,0,0,0,64,0,0,0,0,0,0,4099,4241,659,666
 INSERT INTO `synth_recipes` VALUES (13026,0,0,0,65,0,0,0,0,0,0,4098,4240,657,664,0,0,0,0,0,0,17303,17303,17303,17303,33,66,99,99,'Manji Shuriken');
--- INSERT INTO `synth_recipes` VALUES (13027,0,0,255,65,0,0,0,0,0,0,4096,4238,654,654,716,2764,0,0,0,0,17966,17966,17966,17966,1,1,1,1,'Erlking\'s Tabar');
+INSERT INTO `synth_recipes` VALUES (13027,0,0,23,65,0,0,0,0,0,0,4096,4238,654,654,716,2764,0,0,0,0,17966,17966,17966,17966,1,1,1,1,'Erlking\'s Tabar'); -- JPWiki, others ??, old era talk: success 1st try 21WW
 INSERT INTO `synth_recipes` VALUES (13028,0,0,0,65,0,0,0,0,0,0,4096,4238,2000,2764,0,0,0,0,0,0,19115,19115,19115,19115,1,1,1,1,'Fane Baselard');
--- INSERT INTO `synth_recipes` VALUES (13029,0,0,0,65,0,255,255,0,0,0,4099,4241,666,879,2152,2166,2288,0,0,0,15689,15694,15694,15694,1,1,1,1,'Jaridah Nails');
+INSERT INTO `synth_recipes` VALUES (13029,0,0,0,66,0,28,39,0,0,0,4099,4241,666,879,2152,2166,2288,0,0,0,15689,15694,15694,15694,1,1,1,1,'Jaridah Nails'); -- JPWiki/ffxiclopedia, others 65SM and ??subs
 INSERT INTO `synth_recipes` VALUES (13030,0,0,0,65,0,0,0,0,0,0,4096,4238,8831,0,0,0,0,0,0,0,16476,16476,16476,16476,1,1,1,1,'Darksteel Kukri');
 INSERT INTO `synth_recipes` VALUES (13031,0,0,0,66,0,0,0,0,0,0,4099,4241,662,816,0,0,0,0,0,0,13111,13124,13124,13124,1,1,1,1,'Nodowa');
 INSERT INTO `synth_recipes` VALUES (13032,0,0,0,66,0,0,27,0,0,0,4099,4241,682,682,826,851,851,0,0,0,12811,14209,14209,14209,1,1,1,1,'Darksteel Breeches');
 INSERT INTO `synth_recipes` VALUES (13033,0,1993,0,66,0,0,0,0,0,0,4096,4238,2230,2233,16902,0,0,0,0,0,18427,18427,18427,18427,1,1,1,1,'Hanafubuki');
--- INSERT INTO `synth_recipes` VALUES (13034,0,0,0,66,255,0,0,0,0,0,4096,4238,648,652,654,716,0,0,0,0,17267,17268,17268,17268,1,1,1,1,'Negoroshiki');
+INSERT INTO `synth_recipes` VALUES (13034,0,0,30,66,9,0,0,0,0,0,4096,4238,648,652,654,716,0,0,0,0,17267,17268,17268,17268,1,1,1,1,'Negoroshiki'); -- JPWiki/ffxiclopedia, others ??subs
 INSERT INTO `synth_recipes` VALUES (13035,0,0,0,67,0,0,0,0,0,0,4096,4238,643,643,643,2763,0,0,0,0,2764,2764,2764,2764,1,1,1,1,'Dweomer Steel');
 INSERT INTO `synth_recipes` VALUES (13036,0,0,0,67,0,46,0,0,0,0,4096,4238,662,662,680,680,816,829,0,0,12715,13996,13996,13996,1,1,1,1,'Kote'); -- JPWiki 46CL, mult. talk pages imply 42-48
 INSERT INTO `synth_recipes` VALUES (13037,0,0,0,67,0,0,0,0,0,0,4096,4238,648,662,664,682,850,0,0,0,12427,12439,12439,12439,1,1,1,1,'Bascinet');
 INSERT INTO `synth_recipes` VALUES (13038,0,0,37,67,16,0,0,0,0,0,4096,4238,652,654,719,719,744,0,0,0,18704,18705,18705,18705,1,1,1,1,'Darksteel Hexagun');
--- INSERT INTO `synth_recipes` VALUES (13039,0,0,0,67,255,0,0,0,0,0,4096,4238,654,719,719,744,2764,0,0,0,18737,18737,18737,18737,1,1,1,1,'Fane Hexagun');
+INSERT INTO `synth_recipes` VALUES (13039,0,037,67,16,0,0,0,0,0,4096,4238,654,719,719,744,2764,0,0,0,18737,18737,18737,18737,1,1,1,1,'Fane Hexagun'); -- JPWiki used, others 60+,67SM, 31-45??WW, ??GS
```

Comment:
Missing a comma here. 037 should probably be 0,37

Reply:
oops... got a little happy with the backspace key.  submitted correction while disabling armor plates III/IV, as JPWikki is not actually citing 34AL on those two yet.

Reply:
Could you correct those extra spaces while you are at it? We don't wanna trigger OCD

Reply:
> Could you correct those extra spaces while you are at it? We don't wanna trigger OCD

WHO SUMMONS ME?



:)